AirCasting/SessionViews/ThresholdSettings/ThresholdsSettingsView.swift

```diff
@@ -50,37 +53,22 @@ struct HeatmapSettingsView: View {
                 .buttonStyle(BlueButtonStyle())
                 
                 Button(Strings.SessionCart.resetChangesButton, action: {
+                    thresholdValues = thresholdSettingsViewModel.resetToDefault()
                     presentationMode.wrappedValue.dismiss()
                 })
-                .frame(minHeight: 35)
+                    .frame(minHeight: 35)
             }
             .buttonStyle(BorderlessButtonStyle())
         }
         .onAppear {
-            thresholdVeryLow = "\(Int(changedThresholdValues[0]))"
-            thresholdLow = "\(Int(changedThresholdValues[1]))"
-            thresholdMedium = "\(Int(changedThresholdValues[2]))"
-            thresholdHigh = "\(Int(changedThresholdValues[3]))"
-            thresholdVeryHigh = "\(Int(changedThresholdValues[4]))"
+            thresholdSettingsViewModel.thresholdVeryLow = "\(Int(thresholdValues[0]))"
+            thresholdSettingsViewModel.thresholdLow = "\(Int(thresholdValues[1]))"
+            thresholdSettingsViewModel.thresholdMedium = "\(Int(thresholdValues[2]))"
+            thresholdSettingsViewModel.thresholdHigh = "\(Int(thresholdValues[3]))"
+            thresholdSettingsViewModel.thresholdVeryHigh = "\(Int(thresholdValues[4]))"
```

Comment:
What would you say about sth like:

` .onAppear {
            thresholdSettingsViewModel.thresholdVeryLow = string(using: thresholdValues[0])
            thresholdSettingsViewModel.thresholdLow = string(using: thresholdValues[1])
            thresholdSettingsViewModel.thresholdMedium = string(using: thresholdValues[2])
            thresholdSettingsViewModel.thresholdHigh = string(using: thresholdValues[3])
            thresholdSettingsViewModel.thresholdVeryHigh = string(using: thresholdValues[4])
        }
    }
    
    func string(using thresholdValue: Float) -> String {
        return String(Int(thresholdValue))
    }`
    
    Just to consider: if not, then go below 😇

Reply:
Sure! tbh I didn't give it much thought - just changed this code to use the view model.